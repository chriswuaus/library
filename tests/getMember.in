MEMBER 1000000
add member beachboys
member 111111
member 100000
exit
