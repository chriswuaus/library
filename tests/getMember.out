user: No members in system.

user: Success.

user: No such member in system.

user: 100000: beachboys

user: Ending Library process.
